quipucords/quipucords
# Add a way to verify the authenticity of reports

Closes #2018 
- [x] Added a SHA256SUM file that contains a data integrity hash for each report when downloaded through the `reports/{id} endpoint` 

To test the server side: 
1. Run a scan 
2. Download the tar.gz file containing all reports - note that their is a new file that contains hashes in the tar.gz for each report. Unzip the report - and go to the report directory.
3. Since we are all on macs - run the following command in the terminal 
```shasum -a 256 -c SHA256SUM``` to verify that all the hashes are good 
4. Open 1 or more reports and change data then rerun the above command to verify that a warning is thrown.
